### PR TITLE
Changes for 2.0.0-rc.2

### PR DESCRIPTION
**Pull request by deregtd**
* Bumped all dependencies in ReactXP, all extensions, and all samples to latest versions using npm-check-updates
* Got RXPTest building again on web and ios.
* Fixing regression in web scrollview from a couple weeks ago
* Eliminating setMaxContentSizeMultiplier since it's not in public RN
* Fixing the image api test to take pixel ratio into account
* Updated all versions of extensions to rc.2 (or rc.3 in a few spots) and version of reactxp to rc.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,21 +18,21 @@
     "subscribableevent": "^1.0.1"
   },
   "peerDependencies": {
-    "react": "^16.0",
-    "react-dom": "^16.0",
+    "react": "^16.3",
+    "react-dom": "^16.3",
     "react-native": ">=0.57 <0.60",
```
Comment (mikehardy) [inline package.json]: My only question about this PR (and the release path to 2.x) is why not loose the react-native dependency band to 0.60+ (I'm using it on 0.61 without issue and don't see anything in 0.62 that will break it now that netinfo and webview are standing up nicely with current lean-core extracted repos)

I see the peerDep set to <0.60 for the main module here and all the extensions and it seems like noise in the package install process when they seemingly work fine up to 0.61 at minimum

This is a separate statement from "the samples should be upgraded to RN0.6x" as that's a not fun chore but is orthogonal from the peerDependency setting I think

**Comment by mikehardy** [comment]
Do I smell a release coming :-) ? FWIW I'm effectively using master (2.0.0-rc.1 plus a patch-package patch built from the diff to master) right now with RN 0.61-current as well as a clean 'yarn outdated' for a web build and it all seems fine. 👍

File: package.json (hunk 1 above)
Comment (deregtd) [reply package.json]: Good question... I coulda sworn there was some work left to do to make it compatible with 0.60+ but maybe it was just the samples that needed the chore and we're good.

**Comment by deregtd** [comment]
> Do I smell a release coming :-) ? FWIW I'm effectively using master (2.0.0-rc.1 plus a patch-package patch built from the diff to master) right now with RN 0.61-current as well as a clean 'yarn outdated' for a web build and it all seems fine. 👍

Thanks much for trying this out!  Appreciate the validation. :)

File: package.json (hunk 1 above)
Comment (mikehardy) [reply package.json]: This was my list as I brought my work project up on it - seemed like it was just non-blocking things now (React lifecycle items etc) https://github.com/microsoft/reactxp/issues/1123#issuecomment-520096927

**Comment by mikehardy** [comment]
> Thanks much for trying this out! Appreciate the validation. :)

Integrated rc2 this morning via regular install and was able to evict my patch-package patch. I'm no longer carrying any reactxp-related patches with the current RCs (including ReSub 2.0.0-rc2) with react-native current stable 0.61.4 and I confirm it all seems to work. Have a great weekend

**Comment by deregtd** [comment]
Hoping for a little more testing and feedback from others, then we can do a 2.0.0 final release of everything (reactxp and resub, at least.)

**Pull request merged**